### ISSUE TITLE
KeyError: 'userinfo_uri'` error on 2.0.0

### ISSUE THREAD

**Issue by willthetanman** [fedora-infra/flask-oidc#13]
After updating to 2.0.0 we are encountering this application error

```    from app import app  # noqa: F401
  File "/usr/src/app/app/__init__.py", line 32, in <module>
    oidc = OpenIDConnect(app)
  File "/usr/local/lib/python3.10/dist-packages/flask_oidc/__init__.py", line 111, in __init__
    self.init_app(app, prefix=prefix)
  File "/usr/local/lib/python3.10/dist-packages/flask_oidc/__init__.py", line 136, in init_app
    app.config.setdefault("OIDC_USERINFO_URL", self.client_secrets["userinfo_uri"])
KeyError: 'userinfo_uri'

**Comment by willthetanman** [comment]
https://github.com/fedora-infra/flask-oidc/blob/0fc1eda35123e35c3487a0fedb95cfa789b46d2e/flask_oidc/__init__.py#L136C13-L136C13

It appears "userinfo_uri" is now a required field, this is causing our applications to break which previously did not require this.

**Comment by willthetanman** [comment]
https://github.com/fedora-infra/flask-oidc/blob/0fc1eda35123e35c3487a0fedb95cfa789b46d2e/flask_oidc/__init__.py#L136

It appears this has been added forcing a required "userinfo_uri" in client secrets causing our application to break, previously in 1.4.0 this was not required.

**Comment by abompard** [comment]
Fixed by #14 which is in version 2.0.1 published as of a few minutes ago. Sorry about that.

**Comment by willthetanman** [comment]
Thank you so much for the lightening fast fix!
